requirements.txt

```diff
@@ -18,7 +17,6 @@ jupyterlab
 jupyter-server-proxy
 beautifulsoup4
 lxml
-setproctitle
```

Comment:
I don't know why I added this, but if it appears to work without it, great.

Reply:
Thank you!